src/sentry/integrations/jira/integration.py

```diff
@@ -379,7 +381,7 @@ def get_config_data(self):
         sync_status_forward = {}
 
         if features.has("organizations:jira-per-project-statuses", self.organization):
-            project_mappings = self._filter_active_projects(project_mappings)
+            project_mappings = self._filter_active_projects(list(project_mappings))
```

Comment:
rather than an unnecessary copy the function should be updated to take `Iterable[IntegrationExternalProject]` rather than the incorrect `Sequence` it has now (or you could go concrete and take `QuerySet[IntegrationExternalProject]` -- up to you!)